site/src/contexts/auth/permissions.tsx

```diff
@@ -116,7 +126,6 @@ export const permissionsToCheck = {
 	[checks.viewAnyGroup]: {
 		object: {
 			resource_type: "group",
-			org_id: "any",
```

Comment:
Ended up deleting this because the property wasn't compatible. It also didn't seem to be used anywhere

@Emyrk Not sure if we should update the types so that this can be added back

Reply:
Can you delete this unused check then?

Reply:
Oh, sorry – I meant that the `org_id` property specifically didn't seem to be used anywhere